internal/target/repository.go

```diff
@@ -537,28 +630,115 @@ func (r *Repository) UpdateTarget(ctx context.Context, target Target, version ui
 			"SessionMaxSeconds":      target.GetSessionMaxSeconds(),
 			"SessionConnectionLimit": target.GetSessionConnectionLimit(),
 			"WorkerFilter":           target.GetWorkerFilter(),
+			"Address":                target.GetAddress(),
 		},
 		fieldMaskPaths,
 		[]string{"SessionMaxSeconds", "SessionConnectionLimit"},
 	)
 	if len(dbMask) == 0 && len(nullFields) == 0 {
 		return nil, nil, nil, db.NoRowsAffected, errors.New(ctx, errors.EmptyFieldMask, op, "empty field mask")
 	}
-	var returnedTarget Target
+
+	var filteredDbMask, filteredNullFields []string
+	for _, f := range dbMask {
+		switch {
+		case strings.EqualFold("Address", f):
+		default:
+			filteredDbMask = append(filteredDbMask, f)
+		}
+	}
+	for _, f := range nullFields {
+		switch {
+		case strings.EqualFold("Address", f):
+			address = "null"
```

Comment:
I changed this behavior to set two different boolean flags instead of setting the address variable to "null". The boolean flags are set when the Address field appears in the nullField or the maskField. Do you like this approach or do you prefer something else?